tests/e2e/specs/in-game/gameOver.spec.js

```diff
@@ -7,18 +7,48 @@ const dayjs = require('dayjs');
 function assertVictory() {
   cy.log('Asserting player victory');
   cy.get('#game-over-dialog').should('be.visible').get('[data-cy=victory-heading]').should('be.visible');
+  cy.window()
+    .its('cuttle.app.config.globalProperties.$store.state.game')
+    .then((game) => {
+      if (game.isRanked)
+        cy.get('#game-over-dialog')
+          .should('be.visible')
+          .get('[data-cy=match-result-section]')
+          .should('be.visible');
+      else cy.get('#game-over-dialog').should('be.visible').should('not.contain', 'Match against');
+    });
 }
 
 function assertLoss() {
   cy.log('Asserting player loss');
   cy.get('#game-over-dialog').should('be.visible').get('[data-cy=loss-heading]').should('be.visible');
   cy.get('[data-cy=loss-img]').should('be.visible');
+  cy.window()
+    .its('cuttle.app.config.globalProperties.$store.state.game')
+    .then((game) => {
+      if (game.isRanked)
+        cy.get('#game-over-dialog')
+          .should('be.visible')
+          .get('[data-cy=match-result-section]')
+          .should('be.visible');
+      else cy.get('#game-over-dialog').should('be.visible').should('not.contain', 'Match against');
+    });
 }
 
 function assertStalemate() {
   cy.log('Asserting stalemate');
   cy.get('#game-over-dialog').should('be.visible').get('[data-cy=stalemate-heading]').should('be.visible');
   cy.get('[data-cy=stalemate-img]').should('be.visible');
+  cy.window()
+    .its('cuttle.app.config.globalProperties.$store.state.game')
+    .then((game) => {
+      if (game.isRanked)
+        cy.get('#game-over-dialog')
+          .should('be.visible')
+          .get('[data-cy=match-result-section]')
+          .should('be.visible');
+      else cy.get('#game-over-dialog').should('be.visible').should('not.contain', 'Match against');
```

Comment:
```suggestion
      if (game.isRanked) {
        cy.get('#game-over-dialog')
          .should('be.visible')
          .get('[data-cy=match-result-section]')
          .should('be.visible');
      } else cy.get('#game-over-dialog').should('be.visible').should('not.contain', 'Match against');
```

Reply:
should we add a lint rule to always use curly brackets even if there's only one statement for each condition :) ?

Reply:
Yeah we can do a linting pass when I add some more integrated rules.